03_backend_developement/01_HTTP/01_get_the_weather/.__tests__/04_brussels_weather_in_french.test.js

```diff
@@ -6,6 +6,10 @@ const commandFile = () =>
   path.resolve(__dirname, "../04_brussels_weather_in_french.sh");
 const resultFile = () => path.resolve(__dirname, "../brussels.weather");
 
+afterEach(() => {
+  fs.unlinkSync(resultFile());
```

Comment:
shouldn't we test that the file exists before?

I just tried and `fs.unlinkSync` raise an exception if the files is not here

Reply:
(Same suggestion for each file)

Reply:
You were right, I updated them